samtranslator/model/eventsources/push.py

```diff
@@ -486,13 +486,15 @@ class SNS(PushEventSource):
         "Topic": PropertyType(True, IS_STR),
         "Region": PropertyType(False, IS_STR),
         "FilterPolicy": PropertyType(False, dict_of(IS_STR, list_of(one_of(IS_STR, IS_DICT)))),
+        "FilterPolicyScope": PropertyType(False, IS_STR),
```

Comment:
Avoid potential bugs in the future (see https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#development-guidelines; it's a pass-through value and we don't control the underlying resource):

```suggestion
        "FilterPolicyScope": PassThroughProperty(False),
```

Reply:
added suggested change, thank you!

Reply:
this change broke `make test`. I don't see PassThroughProperty being used elsewhere as well.

Reply:
Ah, please import it on line 9:

```
from samtranslator.model import PropertyType, PassThroughProperty, ResourceMacro
```

Reply:
imported. ran `make test` as well. thank you!